samples/jvm/junit5/README.md

```diff
@@ -0,0 +1,14 @@
+# Sample JS project: Atrium with Junit5
+
+This example project is based on [kotlin's example](https://github.com/Kotlin/kotlin-examples/tree/master/gradle/hello-world).
+Yet with a few optimizations and adaptations:
```

Comment:
I guess you copied this text from the JS examples. I think it does not apply to your sample project